contracts/burner/src/contract.rs

```diff
@@ -39,19 +38,18 @@ pub fn migrate(deps: DepsMut, env: Env, msg: MigrateMsg) -> StdResult<Response>
 
     let data_msg = format!("burnt {} keys", count).into_bytes();
 
-    Ok(Response {
-        messages: vec![SubMsg::new(send)],
-        attributes: vec![attr("action", "burn"), attr("payout", msg.payout)],
-        events: vec![],
-        data: Some(data_msg.into()),
-    })
+    Ok(Response::new()
+        .add_message(send)
+        .add_attribute(("action", "burn"))
+        .add_attribute(("payout", msg.payout))
```

Comment:
Why the embedded tuple?

`.add_attribute("payout", msg.payout)` would read cleaner. Unless you try to accept a variable number of args in a slice (which it seems like you aren't)

Reply:
Yes, so I did the `impl Into<Attribute>` thing just in case someone already has a constructed `Attribute` struct and wants to add it with `add_attribute`. Basically, now this is possible:

```rs
let attr = attr("something", "nice");
// do something else with attr
Ok(Response::new().add_attribute(attr))
```

Is this useful at all? If it'll never happen, I can scratch it.

Reply:
Honestly, I don't see a use of that if the `add_attribute` method takes the same arguments.
The `add_attributes` method does work in the more realistic case that they made a vec of attribute

```rust
let attrs = vec![
   attr("method", "foo"),
   attr("sender", &info.sender),
];
Response::new().add_attributes(attrs);
```

That should work and cover the use case you envision